### PR TITLE
BM-226 restructure

### PR DESCRIPTION
**Pull request by EyesOnlyNet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ Finally: you do not have to create the dist files!
 
 You have to create new versions - the recommended way: `npm version <version-number>`!
 This process will create the dist files and update the version of the library.
-We use semantic versioning - think about it before setting it!
+We use semantic version - think about it before setting it!
```
Comment (tarleb) [inline CONTRIBUTING.md]: The previous wording sounded better to my ears.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Booking Manager Connector
+# === Booking Manager Connector ===
```
Comment (tarleb) [inline README.md:1]: The equal signs will be rendered verbatim on GitHub. Is that intended?

**Review by tarleb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (EyesOnlyNet) [reply README.md:1]: Yes it should really look like "=== ... ===" - encapsulated in "=" signs.

File: CONTRIBUTING.md (hunk 1 above)
Comment (EyesOnlyNet) [reply CONTRIBUTING.md]: i will change it back

**Pull request merged**